.github/workflows/pr.yml

```diff
@@ -269,3 +269,48 @@ jobs:
             --branch-name "${{ github.head_ref || github.ref_name }}" \
             --commit-hash "${{ github.sha }}" \
             --author "${{ github.event.pull_request.user.login || github.actor }}"
+
+  low-memory:
+    name: Low Memory
+    strategy: 
+      matrix:
```

Comment:
instead of family can you make sure to specify an exact instance? otherwise it won't use the exact amount of RAM?

Reply:
are you sure? i accidentally left in a print statement that says the amount of available system memory, and it looks like for all of the ci tests that just ran they ended up getting matched with the correct instance.

think it might be clearer this way that we're targeting machines with 16, 32, 64 gb memory